src/sdk/session/session.ts

```diff
@@ -0,0 +1,70 @@
+import * as jose from "jose";
+import { RemoteJWKSetOptions} from "jose";
+import {User} from "./user";
+
+export default class Session {
+
+    #shortSessionCookieName: string
+    #issuer: string
+    #jwks: any
+
+    constructor(version: string, projectID: string, frontendAPI: string, shortSessionCookieName: string, issuer: string, cacheMaxAge: number) {
+        this.#shortSessionCookieName = shortSessionCookieName
+        this.#issuer = issuer
+
+        const jwkSetUrl = new URL(frontendAPI + "/.well-known/jwks")
+        const joseOptions = new class implements RemoteJWKSetOptions {
+            cacheMaxAge: number = cacheMaxAge;
```

Comment:
so we have a "client side" JWKS cache right? Where is it getting cached, "in process" ?